scripts/paper/generate_stwo_primitive_lookup_vs_naive_figure.py

```
@@ -0,0 +1,299 @@
+
+    return "\n".join(svg)
+
+
+def render_svg(rows: list[dict[str, str]]) -> str:
+    proof_panel = draw_panel(
+        rows,
+        x0=LEFT_X,
+        title="Proof size",
+        metric_key="proof_bytes",
+        metric_label="Estimated raw STARK proof bytes",
+        formatter=lambda value, axis=False: f"{int(value):,}" if not axis else f"{int(value):,}",
```

Ruff (0.15.10)

[warning] Useless `if`-`else` condition

(RUF034)